### DISCUSSION TITLE
[Feature Request] Force 1080p and 720p resolutions

### DISCUSSION THREAD

**Discussion by andrea-scape** [ChristopherKlay/StadiaEnhanced#65] [Ideas & Requests]
I think it would be useful to be able to force lower resolutions than 1440p. Sometimes when i have most of my family connected to internet the game switches back and forth very frequently from 1080p to 720p, it really bothers me beacause it's very distracting.

i tried another extensions called Stadia+ that has this feature but i find Stadia Enhanced vastly better, so i would really appreciate if you implement this feauture.

Keep up the good work!

**Comment by ChristopherKlay** [comment]
While i would love to provide this, i don't think it's currently possible to reliably switch.

Stadia+ has the same issue sometimes, where switching to 720p sometimes still starts in higher resolutions and if it does start in 720p, it sometimes sets your max resolution in Stadias own settings to 720p as well, which stops higher resolutions from working until you manually change that back again.

I'm definitely checking it again however.
